patches/minecraft/net/minecraft/advancements/AdvancementRewards.java.patch

```diff
@@ -9,3 +32,57 @@
        boolean flag = false;
  
        for(ResourceLocation resourcelocation : this.f_9980_) {
+@@ -68,7 +_,7 @@
+       if (this.f_9981_.length > 0) {
+          p_9990_.m_7902_(this.f_9981_);
+       }
+-
++      net.minecraftforge.common.advancements.CustomAdvancementRewards.grant(p_9990_, this.customRewards);
```

Comment:
More context than the player might be nice, such as the lootcontext.

Reply:
I don't think this is needed as that information comes from the player anyway

Reply:
I was thinking for convenience, but idk how often you'd even need the lootcontext.